rotkehlchen/chain/ethereum/modules/ens/decoder.py

```diff
@@ -153,11 +166,108 @@ def _decode_name_renewed(
 
         return None, []
 
+    def _decode_ens_registry_with_fallback_event(
+            self,
+            tx_log: EthereumTxReceiptLog,
+            transaction: EvmTransaction,
+            decoded_events: List[HistoryBaseEntry],
+            all_logs: List[EthereumTxReceiptLog],  # pylint: disable=unused-argument
+            action_items: Optional[List[ActionItem]],  # pylint: disable=unused-argument
+    ) -> Tuple[Optional[HistoryBaseEntry], List[ActionItem]]:
+        if tx_log.topics[0] == NEW_RESOLVER:
+            node = tx_log.topics[1]
+            ens_name = ENS_REVERSE_RESOLVER.call(
+                manager=self.ethereum_manager,
+                method_name='name',
+                arguments=[node],
+            )
+            if ens_name == '':
+                # By checking the contract code, I don't think it can happen. But just in case.
+                return None, []
+
+            # Not able to give more info to the user such as address that was set since
+            # we don't have historical info and event doesn't provide it
+            notes = f'Set ENS address for {ens_name}'
+            decoded_events.append(HistoryBaseEntry(
+                event_identifier=transaction.tx_hash,
+                sequence_index=self.base.get_sequence_index(tx_log),
+                timestamp=ts_sec_to_ms(transaction.timestamp),
+                location=Location.BLOCKCHAIN,
+                location_label=transaction.from_address,
+                asset=A_ETH,
+                balance=Balance(),
```

Comment:
The asset field feels odd here. How does this render in the ui?

Reply:
I am not sure what you are talking about. It renders as an informational event. But what do you suggest to put instead `ETH`? Do you have a better idea?

Reply:
No, just asking if in the UI it is showing an event of 0 ETH or it has a custom format

Reply:
This is what I wanted you to check 
![image](https://user-images.githubusercontent.com/5068010/199091518-054dd80e-2d4c-47b3-bf71-574f7be53c43.png)

Also can you check why this one has a type SWAP? The address is this one

https://etherscan.io/address/0x20Ec6EE26a33d83B57DCdE2859C03C71cf17b5fb

Reply:
https://github.com/rotki/rotki/issues/5066

Reply:
I guess it is a swap due to how frontend displays trades

Reply:
yes exactly. And if you think about it it's not totally wrong, you buy the name and get an ERC721 ENS token.

But perhaps it should not count as a swap. Also ... we should think how should accounting count it.

Cause you can buy a name and then sell it and make profit.